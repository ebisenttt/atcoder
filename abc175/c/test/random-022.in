600807898295830 490511961628148 144422759041924
