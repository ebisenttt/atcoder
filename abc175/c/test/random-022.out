23116862128134
